data/campaigns/Northern_Rebirth/scenarios/09a_Introductions.cfg

```diff
@@ -292,22 +292,22 @@
 
                 [message]
                     speaker=Tallin
-                    message= _ "As Elenia has said, we are currently locked in a death-struggle with the orcs led by a warlord named Rakshas. You could help us most by joining us in our quest to crush this menace once and for all."
+                    message= _ "As Elenia has said, we are currently besieged by  the orcs under a warlord named Rakshas. You could help us most by joining us in our quest to crush this menace once and for all."
                 [/message]
 
                 [message]
                     speaker=Eryssa
-                    message= _ "That could hardly count as repayment of our debt to you, Tallin. This Rakshas fellow is an enemy to every Northern Elf as well. Why, it can’t have been more than a few months ago that the orcs attacked our forests and took me prisoner. Helping you defeat them would be serving our own interests as much as yours."
+                    message= _ "That could hardly count as repayment of our debt to you, Tallin. This Rakshas is an enemy to every northern elf as well. Why, it can’t have been more than a few months ago that the orcs attacked our forests and took me prisoner. Helping you defeat them would be serving our own interests as much as yours."
```

Comment:
In Wesnoth lore, Northern Elf is a proper noun. Refer line 290, for example.

Reply:
Got it.